pkg/blockatlas/client.go

```diff
@@ -57,14 +57,15 @@ func (r *Request) Post(result interface{}, path string, body interface{}) error
 		return err
 	}
 	uri := r.GetBase(path)
-	return r.Execute("POST", uri, buf, result)
+	return r.Execute("POST", uri, buf, result, errors.Params{"path": path})
 }
 
-func (r *Request) Execute(method string, url string, body io.Reader, result interface{}) error {
+func (r *Request) Execute(method string, url string, body io.Reader, result interface{}, params errors.Params) error {
+	params["method"] = method
 	start := time.Now()
 	req, err := http.NewRequest(method, url, body)
 	if err != nil {
-		return errors.E(err, errors.TypePlatformRequest, errors.Params{"url": url, "method": method})
+		return errors.E(err, errors.TypePlatformRequest, params)
```

Comment:
how does this actually address the leaking?
Looking at #837 it seems like the path might be sensitive as well

Reply:
the path is already inside the code, the parameters and the base URL are sensitive

Reply:
Ok thx for explaining.

minor and not sure if adhering to code standards: but to avoid regressions here pls consider adding a brief comment mentioning why we're only passing the `path` and not the full `uri` to the consumer